lesson-3/src/main/java/lesson3/controllers/HelloController.java

```diff
@@ -10,6 +10,11 @@
 
     @GetMapping
     public String getHello() {
-        return "Hello!!!";
+        String message = null;
+        if (true) {
+            System.out.printf("文字数" + message.length());
```

Comment:
\[Checkstyle\] WARNING: 'if' child have incorrect indentation level 12, expected level should be 6\.